### PR TITLE
Remaining commits for OpenOOD v1.5

### PR DESCRIPTION
**Pull request by zjysteven**
@Jingkang50 Here are all of the remaining commits from my side. The detailed change log can be found in the wiki. Notable outcomes include:

- More comprehensive results on CIFAR-10, CIFAR-100, ImageNet-200 (OOD and FS-OOD), and ImageNet-1K (OOD and FS-OOD). For ImageNet-1K we have results on ResNet50, Swin-T, and ViT-B-16.
- An easy-to-use evaluator where people only need to provide a pre-trained classifier, specify the corresponding ID dataset, and specify a postprocessor. There is a notebook tutorial `example_imagenet_eval.ipynb` on this, allowing people who are unfamiliar with OOD detection to test their models with minimum effort.
- OpenOOD can now be installed by `pip install git+https://github.com/Jingkang50/OpenOOD.git`

I expect some updates will be needed for the Readme later.

### CODE REVIEW

File: openood/evaluators/ood_evaluator.py [hunk 1]
```diff
@@ -25,34 +25,56 @@ def __init__(self, config: Config):
...
                 subnet.eval()
         else:
             net.eval()
-        # load training in-distribution data
-        assert 'test' in id_data_loader, \
+        assert 'test' in id_data_loaders, \
             'id_data_loaders should have the key: test!'
         dataset_name = self.config.dataset.name
+
+        if self.config.postprocessor.APS_mode:
```
Comment (zjysteven) [inline openood/evaluators/ood_evaluator.py:43]: @ALin

Comment (2454511550Lin) [reply openood/evaluators/ood_evaluator.py:43]: Sounds good. Thank you for your work!

Comment (2454511550Lin) [reply openood/evaluators/ood_evaluator.py:43]: It looks like in the wiki doc, the leaderboard webpage is not on yet. Any plans for releasing that? That would be really helpful for the community.

Comment (zjysteven) [reply openood/evaluators/ood_evaluator.py:43]: A preview can be found [here](https://zjysteven.github.io/OpenOOD/).

Comment (2454511550Lin) [reply openood/evaluators/ood_evaluator.py:43]: Awesome! Thank you so much!

**Review by Jingkang50 — APPROVED** [review]

**Comment by Jingkang50** [comment]
Hi Jingyang, looks quite awesome to me 👍
Thank you for your GREAT contribution! Feel free to merge!

File: openood/evaluators/ood_evaluator.py (hunk 1 above)
Comment (2454511550Lin) [reply openood/evaluators/ood_evaluator.py:43]: One more question, I think the Tiny ImageNet validation datasets .txt file is not available on one drive right now. Can you share that as well? Thanks!

Comment (zjysteven) [reply openood/evaluators/ood_evaluator.py:43]: The download link to `benchmark_imglist` was updated, and the updated version includes all the new or changed .txt files. Please take a look.
https://github.com/Jingkang50/OpenOOD/blob/f8870d72e3d0abb0c93c123fed7a2d2800f54c2a/scripts/download/download.py#L71

**Pull request merged**